include/stdexec/__detail/__meta.hpp

```diff
@@ -410,6 +421,7 @@ namespace stdexec {
     using __f = _Fn;
   };
 
+  //! Metafunction takes a function and provides a checked version of it?
```

Comment:
The `?` is because I didn't grok this stuff... what's going on with all of these one-letter things?

Reply:
`__q` is like [`mpl::quote`](https://www.boost.org/doc/libs/1_86_0/libs/mpl/doc/refmanual/quote.html). It turns an alias/class template into a metafunction.

it also propagates "meta-exceptions". all of the meta utilities recognize specializations of `stdexec::_ERROR_` as an error type. Error types short-circuit the evaluation of the metafunction and are automatically propagated, like an exception.

this lets me report type errors briefly at the library boundary, even if the actual error happens deep inside a meta-program.